server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/managesieve/ManageSieveMailet.java

```diff
@@ -135,7 +133,7 @@ public void service(Mail mail) throws MessagingException {
 
         // Update the Session for the current mail and execute
         SettableSession session = new SettableSession();
-        if (mail.getAttribute(SMTP_AUTH_USER_ATTRIBUTE_NAME) != null) {
+        if (mail.getAttribute(Mail.SMTP_AUTH_USER_ATTRIBUTE_NAME) != null) {
```

Comment:
I don't think this PR is the place to rethink all the mailet even if I agree with you.